labelbox/schema/project.py

```diff
@@ -79,6 +90,14 @@ class Project(DbObject, Updateable, Deletable):
     benchmarks = Relationship.ToMany("Benchmark", False)
     ontology = Relationship.ToOne("Ontology", True)
 
+    def update(self, **kwargs):
```

Comment:
Is there any danger to toggling the mode? What happens to existing annotations or data in the queue?

Reply:
will be handled by product in the future